pkg/installation/installation.go

```diff
@@ -22,10 +22,11 @@ type Installation struct {
 	Provider          string
 	Codename          string
 	CACert            string
+	SourcePath        string
 }
 
-func New(ctx context.Context, fromUrl string, athenaUrl string) (*Installation, error) {
-	basePath, internalApiPath, err := GetBaseAndInternalPath(fromUrl)
+func New(ctx context.Context, fromUrl, customAthenaUrl string) (*Installation, error) {
+	_, internalApiPath, athenaUrl, err := GetBaseAndInternalPath(fromUrl)
```

Comment:
Added a possibility to provide a custom Athena URL in case it cannot be reliably derived from the Kubernetes API URL